test/helper/contract/ako.go

```diff
@@ -44,6 +45,27 @@ func mustCreateVersionedAtlasClient(ctx context.Context) *admin.APIClient {
 	return client
 }
 
+func mustCreateVersionedAtlasClientSet(ctx context.Context) *atlas.ClientSet {
+	domain := os.Getenv("MCLI_OPS_MANAGER_URL")
+	pubKey := os.Getenv("MCLI_PUBLIC_API_KEY")
+	prvKey := os.Getenv("MCLI_PRIVATE_API_KEY")
```

Comment:
Maybe we could move this to a function to reuse it and not having to call it here and in `newVersionedClient`, th rest is different enough not to to be worth it.

Reply:
I added a function to do this now.

Reply:
@josvazg good to resolve?